classes/controllers/grid/issues/IssueGridHandler.inc.php

```diff
@@ -239,8 +239,7 @@ public function deleteCoverImage($args, $request)
 
         // Check if the passed filename matches the filename for this issue's
         // cover page.
-        $issueDao = DAORegistry::getDAO('IssueDAO'); /* @var $issueDao IssueDAO */
-        $issue = $issueDao->getById((int) $args['issueId']);
+        $issue = Repo::issue()->get((int) $args['issueId']);
```

Comment:
This affected the old code already, but there's no check here to ensure that the user-specified issue ID is in the current journal.